package.json

```diff
@@ -81,20 +79,20 @@
     "webpack-node-externals": "^1.6.0"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.4.0",
     "amazon-cognito-identity-js": "^2.0.1",
     "archiver": "^2.1.1",
-    "aws-sdk": "^2.326.0",
```

Comment:
This was done to save on zip size as it's already provided by the runtime. Think we need some more investigation into best practice around this.